crates/neon/src/types_impl/mod.rs

```diff
@@ -231,12 +318,36 @@ impl private::ValueInternal for JsNull {
     }
 }
 
-/// A JavaScript boolean primitive value.
+/// The type of JavaScript
+/// [Boolean](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Data_structures#primitive_values)
+/// primitives.
+///
+/// # Example
+///
+/// ```
+/// # use neon::prelude::*;
+/// # fn test(mut cx: FunctionContext) -> JsResult<JsUndefined> {
+/// // Extract the console.log function:
+/// let console: Handle<JsObject> = cx.global().get(&mut cx, "console")?;
```

Comment:
This repeated example code makes me think we might want to file an issue to include typed structs for common globals.

```rust
struct Console<'a> { /* ... */ }

impl<'a> Console<'a> {
    pub fn log<'cx, C: Context<'cx>, A: Arguments>(cx: &mut C, args: A) -> NeonResult<()> { todo!() }
}

let console: Console = cx.console()?;
let hello = cx.string("Hello, World!");

console.log(&mut cx, (hello,));
```

Reply:
That's lovely! Other examples I can think of are `process` and `JSON`. I filed #958.